lib/records/fhir.rb

```diff
@@ -56,6 +59,8 @@ def self.basic_info(entity, fhir_record, end_time = Time.now)
                                              'text' => { 'status' => 'generated',
                                                          'div' => '<div xmlns="http://www.w3.org/1999/xhtml">Generated by <a href="https://github.com/synthetichealth/synthea">Synthea</a>. '\
                                                                    "Version identifier: #{Synthea::Config.version_identifier}</div>" },
+                                             # hospital
+                                             'managingOrganization' => { 'reference' => prov.fullUrl.to_s },
```

Comment:
Not sure `managingOrganization` is the best field for this. The description is "Organization that is the custodian of the patient record" but you're really trying to represent the fact that this is where they usually go for health care services. I think the `generalPractitioner` field is more appropriate, even though we're putting a hospital in there not a single doctor.